configstack/module.go

```diff
@@ -553,24 +561,22 @@ func FindWhereWorkingDirIsIncluded(terragruntOptions *options.TerragruntOptions,
 		cfgOptions, err := options.NewTerragruntOptionsWithConfigPath(dir)
 		if err != nil {
 			terragruntOptions.Logger.Debugf("Failed to build terragrunt options from %s %v", dir, err)
-			return nil
+			continue
 		}
 
 		cfgOptions.Env = terragruntOptions.Env
 		cfgOptions.LogLevel = terragruntOptions.LogLevel
 		cfgOptions.OriginalTerragruntConfigPath = terragruntOptions.OriginalTerragruntConfigPath
 
-		if terragruntOptions.TerraformCommand == "destroy" {
-			var hook = NewForceLogLevelHook(logrus.DebugLevel)
-			cfgOptions.Logger.Logger.AddHook(hook)
-		}
+		var hook = NewForceLogLevelHook(logrus.DebugLevel)
+		cfgOptions.Logger.Logger.AddHook(hook)
 
 		stack, err := FindStackInSubfolders(cfgOptions, terragruntConfig)
 		if err != nil {
-			// loggign error as debug since in some cases stack building may fail because parent files can be designed
+			// log error as debug since in some cases stack building may fail because parent files can be designed
 			// to work with relative paths from downstream modules
 			terragruntOptions.Logger.Debugf("Failed to build module stack %v", err)
-			return nil
+			continue
```

Comment:
Due to this change, stack might be nil in 582

Reply:
on line 579 execution will go to next element in for loop, will not be invoked `stack.Modules`

Reply:
You are correct. The for loop was hidden when i reviewed this section.